src/psyclone/tests/domain/lfric/lfric_loop_test.py

```diff
@@ -734,6 +877,7 @@ def test_dynloop_halo_read_access_error2(monkeypatch):
 def test_null_loop():
     ''' Check that we can create a 'null'-type loop and that the validation
     check in the load() method behaves as expected.
+
```

Comment:
I would suggest single quotes around `'load()`' in the line above.

Reply:
done